docker/fullnode/README.md

```diff
@@ -30,6 +29,29 @@ Get the latest version of the Sui DevNet genesis [genesis.blob](https://github.c
 ```wget https://github.com/MystenLabs/sui-genesis/raw/main/devnet/genesis.blob```
 
 
+### docker compose 
+
+Download the latest version of docker compose
+
+```wget https://github.com/MystenLabs/sui/blob/main/docker/fullnode/docker-compose.yaml```
+
+Configure physical resource constraints and logging options
```

Comment:
is this an instruction to modify the yaml file to configure Docker Compose?

Reply:
yes, maybe we should add it directly in docker-compose.yml